ginivision/src/main/java/net/gini/android/vision/camera/CameraActivity.java

```diff
@@ -395,28 +477,44 @@ private void startOnboardingActivity() {
     @Override
     public void onDocumentAvailable(@NonNull Document document) {
         mDocument = document;
-        // Start ReviewActivity
-        mReviewDocumentActivityIntent.putExtra(ReviewActivity.EXTRA_IN_DOCUMENT, document);
-        mReviewDocumentActivityIntent.putExtra(EXTRA_IN_ANALYSIS_ACTIVITY, mAnalyzeDocumentActivityIntent);
-        mReviewDocumentActivityIntent.putExtra(ReviewActivity.EXTRA_IN_BACK_BUTTON_SHOULD_CLOSE_LIBRARY,
-                mBackButtonShouldCloseLibrary);
-        startActivityForResult(mReviewDocumentActivityIntent, REVIEW_DOCUMENT_REQUEST);
+        if (mDocument.isReviewable()) {
```

Comment:
Images are reviewable and it makes sense to show them in the Review Screen, but pdfs are not editable and can be passed to the Analysis Screen directly.